src/app/navbar/navbar.component.css

```diff
@@ -45,7 +45,7 @@
 
 .toolbar-holder-sm {
   height: auto !important;
-  margin: 5px;
+  padding: 5px;
```

Comment:
Why did this change? Can you please add a comment, in the commit message or in the PR explaining why.

Reply:
One PWA is verifying that the content is sized correctly for the viewport. Specifically for mobile viewports.

Margin affects the outer space of an element, which increases the `innerWidth` of the window's content area but not the `outerWidth` of the browser's window. As is, the `innerWidth !== outerWidth` because margin added `5px` to the content area

Padding affects the inner space of an element but does not increase the `innerWidth`, so the viewport content is correctly sized.

Reply:
Thanks for the explanation! I'm aware of the difference between padding and margin in general, but didn't know why this was being done in particular. I think you should add this to the commit message when you merge.

I'm also a little wary that this might introduce some unintended styling changes on a page with the navbar. Do pages visually look the same with and without this change?

Reply:
The difference is noticeable but also pretty negligible. Left is local, right is dockstore.org

<img width="1280" alt="Screen Shot 2019-04-24 at 2 37 37 PM" src="https://user-images.githubusercontent.com/13545729/56695764-afb12980-669e-11e9-9494-b5f697e50415.png">